com.etlegacy.ETLegacy.yaml

```diff
@@ -0,0 +1,191 @@
+app-id: com.etlegacy.ETLegacy
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+branch: stable
+
+# 64bit is currently unfunctional. Upsteam is aware of this,
+# but there is no definitive solution as of yet.
+
+# https://dev.etlegacy.com/issues/1211
+
+# There is also a more controversial problem, relating to mods.
+# Many mods are made under a crappy non-GPL compatible SDK license which
+# means that those mods will forever remain closed source, 32bit only.
+# Good hope is that overtime they will die, but until then, we better stick
+# to 32bit
+
+command: etl.sh
+rename-icon: etl
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=x11
```

Comment:
Is wayland really  used? If so can this be run using fallback-x11 rather than x11?

Reply:
I don't actually have a way to test it. They do use SDL 2 so that should take care of it.

Reply:
I just did test it with `--nosocket=x11` and it fails, 
```
GLimp_Init() - could not load OpenGL subsystem
```
`--nosocket=x11 --socket=fallback-x11` also fails, so I think it is safe to assume that the wayland flag has to be removed.

Reply:
Is good, I'll remove it. Thanks for testing it for me.

Damn. I really need an AMD card now.

Reply:
Ow, what is your test setup?

Reply:
Could it be because of GLU and GLEW? Otherwise, it could well be that Wayland support is only partially implemented. After all, SDL might support it in 2.0.4 but if they don't, then it won't matter.

Shall I remove the --wayland parameter?